client/src/components/Instructions.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+
+const Instructions = () => (
+  <div className="Instructions">
```

Comment:
** [class-wrapper-names] ** We have been trying to use a wrapper syntax for these types of classes. Should we consider that here too?